contrib/endpoints/src/api_manager/config.cc

```diff
@@ -113,6 +113,36 @@ MethodInfoImpl *Config::GetOrCreateMethodInfoImpl(const string &name,
   return i->second.get();
 }
 
+bool Config::LoadQuotaRule(ApiManagerEnvInterface *env) {
```

Comment:
https://cs.corp.google.com/piper///depot/google3/tech/internal/env/framework/wrappers/chemist/chemist_wrapper.cc?rcl=145718473&l=725

According to the source code
- quota_metrics conversion from rules and metric_rules are different. 
- if there is rules, then metric_rules will be ignored. 
- cost field can be empty, it should be 1